core/src/main/scala/org/apache/spark/sql/SnappySession.scala

```diff
@@ -1855,7 +1855,11 @@ object SnappySession extends Logging {
         evaluatePlan(df, session, key.sqlText, key)
       }
     }
```

Comment:
Yeah. Lead processing does not benefit much from the cache lines, I suppose. So, that is ok.